bundles/core/org.eclipse.smarthome.core.thing.test/src/test/groovy/org/eclipse/smarthome/core/thing/firmware/FirmwareRegistryOSGiTest.groovy

```diff
@@ -134,7 +138,7 @@ final class FirmwareRegistryOSGiTest extends OSGiTest {
 
     @After
     void teardown() {
-        Locale.setDefault(defaultLocale)
+        setDefaultLocale(defaultLocale)
         unregisterMocks()
```

Comment:
Not added by yourself in this case, but you do it below, so I added the comment here too.

Isn't `unregisterMocks()` already be called by the test framework caused by the `@After` the method is using?